app/js/components/shared/BreadCrumb/BreadCrumb.jsx

```diff
@@ -0,0 +1,100 @@
+/*
```

Comment:
This component is fully managed by its state, it watches for changes in the URL Path and updates it states accordingly. It's mostly static and I can see it getting quite complex if we start having deeply nested pages e.tc
For now the styling is also minimal.